scripts/salt-setup.sh

```diff
@@ -8,7 +8,7 @@ set -ex -o pipefail -o errexit
 function prepare {
   sudo chown -R root:root /tmp/saltstack
   apply_amazonlinux_salt_patch
-  apply_rhel8_salt_patch
+  #apply_rhel8_salt_patch
```

Comment:
Runtime images are still using salt 3001.8 so I think this patch is still needed. Or we should upgrade salt for runtime images as well.

Reply:
restored